modules/network-observability-quickfilter.adoc

```diff
@@ -7,6 +7,13 @@
 = Filtering the network traffic
 By default, the Network Traffic page displays the traffic flow data in the cluster based on the default filters configured in the `FlowCollector` instance. You can use the filter options to observe the required data by changing the preset filter.
 
+Query Options::
+You can use *Query Options* to optimize the search results, as listed below:
+
+** *Log Type*:  A conversation is an aggregation of flows between the same peers. The available options *Conversation* and *Flows* provide the ability to query flows by events, such as new connection, completed connection, periodic record with updates for long connections and flow logs. 
```

Comment:
Here its mentioned `periodic record with updates` and at later stages its mentioned `Heartbeat`. maybe a uniform word across would be better

Reply:
Hmm, I see your point. I wonder if I change to `events, such as new connection, completed connection, flow logs, and a heartbeat, or periodic record with updates for long connections.` so that heartbeat is there, but it has some connotation.

Reply:
Yeah that looks better. I think all the 3 events relevant to connection tracking be put together and flow logs at the start/end of it but not in between(I know its a lot nitpicking but yeah I am just saying what feels/looks right to me)

Reply:
That makes sense. Agree and addressed.